.github/workflows/wheel.yaml

```diff
@@ -53,7 +43,7 @@ jobs:
           python-version: '3.9'
 
       - name: Build sdist
-        run: python setup.py sdist
+        run: pipx run build --sdist
```

Comment:
Do we need to add `pipx` as a requirement somewhere? Or is that already installed?

Reply:
It's built into the default runners as a supported package manager. See https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md or any other platform under https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-software . It does not require setup-python (that could be moved below this line). (pipx is included in manylinux images, too, which is really handy when you want to run something like nox/tox on many Pythons ;) )